lib/src/components/NxForm/NxForm.tsx

```diff
@@ -79,9 +79,10 @@ const _NxForm = forwardRef<HTMLFormElement, Props>(
             <footer className="nx-footer">
               <NxLoadError titleMessage={submitErrorTitleMessage || 'An error occurred saving data.'}
                            error={submitError}
-                           submitOnRetry={true} />
+                           submitOnRetry={true}
+                           aria-label="form saving errors"/>
               { formHasValidationErrors && !submitError &&
-                <NxErrorAlert className="nx-form__validation-errors">
+                <NxErrorAlert className="nx-form__validation-errors" aria-label="form validation errors">
```

Comment:
I don't see a test for this one

Reply:
> I don't see a test for this one

I thought I added ops: 5fc6469c0dee477ca029534283a2b9bca3fd7633